milestone1.css

```diff
@@ -0,0 +1,55 @@
+.recipes {
+    font-family: "Papyrus";
+    background-color: rgb(219, 199, 245);
+    padding: 10px;
+    margin: 10px;
+    border: 1px rgb(219, 199, 245);
+    border-radius: 10px;
+}
+
+.header {
+    font-family: "Papyrus";
+    background-color: rgb(120, 75, 152);
+    padding: 50px
+    margin: 10px
+    border: rgb(120, 75, 152);
+}
```

Comment:
Maybe add some contrast to the text color so that it stands out and is readable against the purple background